### ISSUE TITLE
Sentiment Analysis on Mastodon Posts -  Predicting Election Outcomes with Elixir? 

### ISSUE THREAD

**Issue by sehHeiden** [elixir-berlin/planning#82]
### Your name

Sebastian Heiden

### Your GitHub

https://github.com/sehHeiden/

### Your Twitter (or other social link)

@Muemmel@chaos.social

### Your proposal type: talk, lightning talk or til (today I learned)

Talk? 

### Title of your presentation

Mastodon Posts -  Predicting Election Outcomes with Elixir? 

### Description

As I study Project I try to analysis the Sentiment in Posts/toots about the Bavaria State Election all in Elixir from crawling the data with httPoison and saving in a SQLite with Ecto, to data Analysis/ Cleaning in Explorer, VegaLite to Analysis with Bumblebee for loading the language models.
Project will be finished End November 2023.

### Favourite Month you would like to present in

February

**Comment by slashmili** [comment]
Hey @sehHeiden! That's an interesting topic and we will be glad to hear from you in the Feb event. The place for Feb is not finalised yet. We'll ping you mid January.

**Comment by sehHeiden** [comment]
Thanks. I just handed in my assignment this talk will be based on yesterday, somewhat delayed because of illness.

The text can be read here:
 https://github.com/sehHeiden/poll_election_bavaria_2023/tree/main

The gist is, that I tried it on the Bavarian state outcome this October. I did have an average error of 4 to 5 percent per party. I will discuss it with my professor. Perhaps we can enhance the result somewhat. But the limited user base of Mastodon is working against use here.

Besides, in the presentation, I could come from a Data Engineering or Analysis perspective. I'll have to see. 

Having exams till end of January and my day job, I hope this meetup is not too early in February.

Hope to see you there.
